src/index.ts

```diff
@@ -55,11 +56,20 @@ async function main() {
         branch
       });
       console.log(`Found ${data.total_count} runs total.`);
+
+      let cancel_before;
+      if (all_but_latest) {
+        cancel_before = new Date(data.workflow_runs.reduce((a, b) => a.created_at < b.created_at ? a.created_at : b.created_at));
```

Comment:
```suggestion
        cancel_before = new Date(data.workflow_runs.reduce((a, b) => Math.min(a.created_at, b.created_at);
```

This should be equivalent. But I'm a bit confused because I thought you want the max date, not the min date (the "latest" would be max, right?)

Reply:
That's a long time ago... I think you're right about `max` though.